src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php

```diff
@@ -5,7 +5,12 @@
 namespace SmartboxSkeletonRemoteDemoBundle\Controller;
 
 use SmartboxSkeletonBundle\Entity\PingMessage;
+<<<<<<< HEAD
```

Comment:
Oh Mel

Reply:
ha ha ha.. how did that slip though!!! I will fix it now :-D